Storage/Rust/RustLogins.swift

```diff
@@ -4,14 +4,13 @@
 
 import Foundation
 import Shared
-import Deferred
 
 @_exported import MozillaAppServices
 
 private let log = Logger.syncLogger
 
 public extension LoginRecord {
-    public convenience init(credentials: URLCredential, protectionSpace: URLProtectionSpace) {
+    convenience init(credentials: URLCredential, protectionSpace: URLProtectionSpace) {
```

Comment:
Why'd you change the accessibility of these? I'm pretty sure these are needed outside of the `Storage` framework.

Reply:
Compiler warnings that say members of 'public extension' are implicitly 'public'.